config/initializers/raven.rb

```diff
@@ -0,0 +1,5 @@
+require 'raven'
+
+Raven.configure do |config|
+  config.dsn = 'https://d7bdb16392f8445c9c96340a4c273eb0:03c7913b6d2d49518dbb28864060054b@app.getsentry.com/44574'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.